ConsoleProject/src/view/Frontend.cs

```diff
@@ -146,7 +170,7 @@ public class Frontend {
             pictureBox.Size = bmp.Size;
         }
 
-        private void LoadImages() {
+        public void LoadImages(string filename) {
```

Comment:
Why did this method had to become public?

Reply:
Mhhh... Forgot to revert it back to Private.
That was when I thought I could interact with it from TextureConverter. Fixed.